server/app.py

```diff
@@ -210,6 +224,88 @@ async def submit_batch(batch: BatchData):
     return Response[BatchSubmitted](data=BatchSubmitted(job_id=batch.job_id))
 
 
+def get_prompt_tokens(string: str, model: str, output_fields: List[str]) -> int:
+    user_tokens = litellm.token_counter(model=model, text=string)
```

Comment:
Of course the endpoint needs to be here but what do you think about putting the litellm stuff in litellm runtime? Seems cleaner to have it all in one file, wdyt? Also not using other things that are part of the runtime here, just one simple function, so not a clear choice to me, wondering if you thought about it

Reply:
yeah, I was on the fence about it. It kinda feels like we don't need the separate litellm runtime since it's sort of subsuming everything else, but I agree that if we have it this probably makes sense to put in there. I'll move it and if we decide to break that up later we can decide the best place for it then.